examples/landmark_uncertainty/main.py

```diff
@@ -0,0 +1,251 @@
+"""
+Uncertainty Estimation for Landmark Localisaition
+
+
+Reference:
+L. A. Schobs, A. J. Swift and H. Lu,
+"Uncertainty Estimation for Heatmap-Based Landmark Localization,"
+in IEEE Transactions on Medical Imaging, vol. 42, no. 4, pp. 1021-1034,
+April 2023, doi: 10.1109/TMI.2022.3222730.
+
+Paper link: https://arxiv.org/abs/2203.02351
+"""
+
+import argparse
+import os
+import shutil
+import warnings
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from config import get_cfg_defaults
+from pandas import *
+
+from kale.embed.uncertainty_fitting import fit_and_predict
+
+warnings.filterwarnings("error")
```

Comment:
Shall we put this after all imports?

Reply:
Not necessarily. If i remember correctly, I put this because of case where numpy warnings were showing, but they were expected for me. From googling around, the best way to suppress them was with this.